admin-dev/themes/new-theme/js/pages/product/edit/specific-prices-manager.ts

```diff
@@ -80,7 +80,7 @@ export default class SpecificPricesManager {
     // Enable/disabled the priority selectors depending on the priority type selected (global or custom)
     new FormFieldDisabler({
       disablingInputSelector: PriorityMap.priorityTypeCheckboxesSelector,
-      matchingValue: '0',
+      matchingValue: '1',
```

Comment:
I would suggest to use a `const` or an `enum` there to make it more verbose and avoid mistakes in the future